src/tests/ftest/util/test_utils_pool.py

```diff
@@ -490,26 +490,45 @@ def wait_for_rebuild(self, to_start, interval=1):
             "Rebuild %s detected", "start" if to_start else "completion")
 
     @fail_on(DaosApiError)
-    def start_rebuild(self, ranks, daos_log):
-        """Kill the specific server ranks using this pool.
+    def start_rebuild(self, ranks, daos_log, use_kill=False):
```

Comment:
Since we're adding a DaosCommand call to this method we should also fail the method on any exception it may raise:
```suggestion
    @fail_on(CommandFailure)
    def start_rebuild(self, ranks, daos_log, use_kill=False):
```

Reply:
Done